Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -120,6 +120,7 @@
   abstract: true
   components:
   - type: CartridgeLoader
+    diskSpace: 7 # Has one extra pre-installed program.
```

Comment:
I think what chromiumboy meant was that all pdas should have the same *total* of 8 slots, not 8 free slots.
So remove this datafield everywhere.